metabase/metabase
# Fix click behavior routing to a different dashboard with a non-default target tab

- Closes: https://github.com/metabase/metabase/issues/39320

---
### How to Manually Verify
1. Create two dashboards
  - (1) Add a card configured with click behavior routing to the second dashboard with the 2nd tab being the target tab
    ![image](https://github.com/metabase/metabase/assets/22608765/1d7468a2-3661-4f02-9f96-8753f0ae9e43)
  - (2) Create a two tabbed dashboard with a filter applied to a card on each tab
    ![image](https://github.com/metabase/metabase/assets/22608765/0220a7b1-05ef-4523-9ac0-54722dce5b0f)
    
2. Test click behavior on the first dashboard and verify that it routes you to the correct dashboard and correct tab
  ![image](https://github.com/metabase/metabase/assets/22608765/dd49bf33-cf25-4adf-bdc6-f8ac0605f4d9)
  ![image](https://github.com/metabase/metabase/assets/22608765/0f966d6a-1425-4aae-815e-733fe3ec4b1e)
4. Test multiple times: try refreshing on the first dashboard and testing it again. Verify that it routes you to the correct tab on every attempt.
